examples/blockfinder.js

```diff
@@ -1,19 +1,23 @@
 /*
  * This simple bot will help you find any block
  */
-const mineflayer = require('mineflayer')
-
-const { performance } = require('perf_hooks')
+import { createBot } from 'mineflayer'
+import { performance } from 'perf_hooks'
 
 if (process.argv.length < 4 || process.argv.length > 6) {
   console.log('Usage : node blockfinder.js <host> <port> [<name>] [<password>]')
   process.exit(1)
 }
 
-const bot = mineflayer.createBot({
+// TODO: commands
+console.log('Commands :\n' +
+  '  loaded\n' +
+  '  starts with find')
```

Comment:
Why console log? It's not intended to be used as a bot. It's intended to act as an example of a bot. Anyone who use that file will look at the code before running it.